<!DOCTYPE HTML PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"><head>


<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<title>NCBI Blast:1smd AMY1_HUMAN</title>
<link rel="stylesheet" type="text/css" href="1smd_vs_1bf2_blast2sequences_files/main.css" media="screen">
<link rel="stylesheet" type="text/css" href="1smd_vs_1bf2_blast2sequences_files/blastRes.css" media="screen">
<!--[if IE]>
<link rel="stylesheet" type="text/css" href="css/descriptionsIE.css" media="screen" />
<![endif]-->
<link rel="stylesheet" type="text/css" href="1smd_vs_1bf2_blast2sequences_files/print.css" media="print">
<!--[if lte IE 6]>
<link rel="stylesheet" type="text/css" href="css/ie6_or_less.css" />
<![endif]-->
<link rel="stylesheet" type="text/css" href="1smd_vs_1bf2_blast2sequences_files/blastReq.css" media="screen">
<link rel="stylesheet" type="text/css" href="1smd_vs_1bf2_blast2sequences_files/results.css" media="screen">
<link rel="stylesheet" id="sugstyle" href="1smd_vs_1bf2_blast2sequences_files/ncbisuggest.css" type="text/css">
</head><body id="type-a" class="noToggleCheck">
<div id="wrap">
			<div id="header">	    
		<div id="site-name"><a id="logolink" href="http://www.ncbi.nlm.nih.gov/" title="NCBI Home Page"><img src="1smd_vs_1bf2_blast2sequences_files/helix.gif" alt="NCBI Logo" title="Link to NCBI Home Page"></a>BLAST <span class="desc"> Basic Local Alignment Search Tool</span>
		</div>
		<div id="search">
		
		</div>		
		<ul id="nav">
                <li class="first"><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastHome">Home</a></li>
                <li class="recent"><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=GetSaved&amp;RECENT_RESULTS=on">Recent Results</a></li>
                <li class="saved"><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=GetSaved">Saved Strategies</a></li>
                <li class="last documentation"> <a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs">Help</a></li>                
                </ul>
    </div>

        <div id="content-wrap">

                <div id="breadcrumb" class="inlineDiv">
                   <a href="http://www.ncbi.nlm.nih.gov/">NCBI</a>/
                   <a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastHome">BLAST</a>/
                   <a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?PAGE=Proteins&amp;PROGRAM=blastp&amp;BLAST_PROGRAMS=blastp&amp;PAGE_TYPE=BlastSearch&amp;SHOW_DEFAULTS=on&amp;BLAST_SPEC=blast2seq">blastp suite-2sequences</a>/
                   <strong>Formatting Results - T25E2RUR11N</strong>             
                </div>
                <div class="inlineDiv resHeader resPage">
				   
                </div>
                
                <!-- CDD info for old design -->
                
                
                


                				
				

                <div id="content">                              		
                                
                <!--<ul id="msg" class="msg"><li class=""><p class=""></p><p class=""></p><p class=""></p></li></ul>-->
                <ul id="msg" class="msg"><li class=""></li></ul>                
                <a href="#" class="resLink resPage" id="searchOptions" submitform="setSavedOptions">Edit and Resubmit</a>
<a href="#" class="resLink resPage hidden" id="saveSearchOptions" submitform="saveOptions">Save Search Strategies</a>
<span class="reslink resPage">[Sign in above to save your search strategy]</span>	
<a id="showFormat" class="resArrowLinkB resLink resPage hiding" href="#">Formatting options </a>
<a id="showDownload" class="resArrowLinkB resLink resPage hiding" href="#">Download </a>
                <form action="Blast.cgi" enctype="application/x-www-form-urlencoded" method="post" id="results">
                <input name="ALIGNMENTS" value="100" type="hidden"><input name="ALIGNMENT_VIEW" value="Pairwise" type="hidden"><input name="BL2SEQ" value="on" type="hidden"><input name="BLAST_INIT" value="blast2seq" type="hidden"><input name="BLAST_PROGRAMS" value="blastp" type="hidden"><input name="BLAST_SPEC" value="blast2seq" type="hidden"><input name="CDD_SEARCH" value="on" type="hidden"><input name="CLIENT" value="web" type="hidden"><input name="COMPOSITION_BASED_STATISTICS" value="2" type="hidden"><input name="DATABASE" value="nr" type="hidden"><input name="DESCRIPTIONS" value="100" type="hidden"><input name="EQ_OP" value="AND" type="hidden"><input name="EXPECT" value="10" type="hidden"><input name="FORMAT_OBJECT" value="Alignment" type="hidden"><input name="FORMAT_TYPE" value="HTML" type="hidden"><input name="FULL_DBNAME" value="n/a" type="hidden"><input name="GAPCOSTS" value="11 1" type="hidden"><input name="GET_SEQUENCE" value="on" type="hidden"><input name="JOB_TITLE" value="1smd AMY1_HUMAN" type="hidden"><input name="LAYOUT" value="OneWindow" type="hidden"><input name="MASK_CHAR" value="2" type="hidden"><input name="MASK_COLOR" value="1" type="hidden"><input name="MATRIX_NAME" value="BLOSUM62" type="hidden"><input name="MAX_NUM_SEQ" value="100" type="hidden"><input name="NUM_DIFFS" value="0" type="hidden"><input name="NUM_OPTS_DIFFS" value="0" type="hidden"><input name="NUM_OVERVIEW" value="100" type="hidden"><input name="OLD_BLAST" value="false" type="hidden"><input name="PAGE" value="Proteins" type="hidden"><input name="PAGE_TYPE" value="BlastSearch" type="hidden"><input name="PROGRAM" value="blastp" type="hidden"><input name="QUERY_INFO" value=" 1smd AMY1_HUMAN (488 letters)" type="hidden"><input name="QUERY_LENGTH" value="488" type="hidden"><input name="REPEATS" value="repeat_9606" type="hidden"><input name="RID" value="T25E2RUR11N" type="hidden"><input name="RTOE" value="1222" type="hidden"><input name="SEARCH_DB_STATUS" value="43" type="hidden"><input name="SELECTED_PROG_TYPE" value="blastp" type="hidden"><input name="SERVICE" value="plain" type="hidden"><input name="SHORT_QUERY_ADJUST" value="on" type="hidden"><input name="SHOW_LINKOUT" value="on" type="hidden"><input name="SHOW_OVERVIEW" value="on" type="hidden"><input name="USER_DEFAULT_MATRIX" value="3" type="hidden"><input name="USER_DEFAULT_PROG_TYPE" value="blastp" type="hidden"><input name="USER_TYPE" value="2" type="hidden"><input name="WORD_SIZE" value="3" type="hidden"><input name="_PGR" value="0" type="hidden"><input name="db" value="protein" type="hidden"><input name="stype" value="protein" type="hidden">
                <input name="QUERY_INDEX" value="0" type="hidden">
                <input name="_PGR" value="0" type="hidden">
                <input name="CMD" value="Get" type="hidden">                
                </form>
                <!-- Reformat and downlaod for new design -->
                <form action="Blast.cgi" enctype="application/x-www-form-urlencoded" method="post" name="FormatForm" id="FormatForm" class="resbox hidden" toggle="showFormat">
<table class="options blastp">
<tbody><tr><th class="frh" colspan="2">
<div class="fi">
Formatting options
<img id="onPageReformat" src="1smd_vs_1bf2_blast2sequences_files/reformat.jpg" class="blastbutton" alt="Reformat" mouseovimg="css/images/reformatOver.jpg" mouseoutimg="css/images/reformat.jpg" mousedownimg="css/images/reformatDown.jpg" mouseupimg="css/images/reformatOver.jpg" border="0">
</div>
</th></tr>
<tr class="paramSet">
<th>Show</th>
<td>
<div class="fi">
<select id="FORMAT_OBJECT" class="reset" name="FORMAT_OBJECT" defval="Alignment">
<option value="Alignment" selected="selected">Alignment</option>
<!--<option value="PSSM" >PSSM</option>
<option value="PSSM_Scoremat" >PssmWithParameters</option>
<option value="Bioseq"  >Bioseq</option>-->
</select>
as
<select name="FORMAT_TYPE" id="FORMAT_TYPE" class="reset" defval="HTML">
<option value="HTML" selected="selected">HTML</option>
<option value="Text">Plain text</option>
<!--<option value="ASN.1"  >ASN.1</option>
<option value="XML"  >XML</option>-->
</select>
<!-- 
<input name="PSSM_FORMAT_TYPE" value="Text" size="3" id="pssmFormat" type="text" class="hidden dispType" />
<input name="BIOSEQ_FORMAT_TYPE" value="ASN.1" size="3" id="bioseqFormat" type="text" class="hidden dispType" />
<input name="PSSM_SC_FORMAT_TYPE" value="ASN.1" size="3" id="pssmScFormat" type="text" class="hidden dispType" />
-->
<span id="advView" class="">
<input name="NEW_VIEW" id="NEW_VIEW" class="cb reset" defval="checked" type="checkbox">
<label for="NEW_VIEW">Advanced View</label>
<span class=""><input name="OLD_BLAST" id="OLD_BLAST" class="cb reset" defval="unchecked" false="" type="checkbox">
<label for="OLD_BLAST">Use old BLAST report format</label></span>
</span>
<a class="resetAll" id="resetAll">Reset form to defaults</a>
<a class="helplink hiding" title="help" id="formatHelp" href="#"><span>[?]</span></a>
<p toggle="formatHelp" class="helpbox hidden" id="hlp1">
These options control formatting of alignments in results pages. The
default is HTML, but other formats (including plain text) are available.
PSSM and PssmWithParameters are representations of Position Specific Scoring Matrices and are only available for PSI-BLAST. 
The Advanced view option allows the database descriptions to be sorted by various indices in a table.
<a href="http://www.ncbi.nlm.nih.gov/BLAST/blastcgihelp.shtml#format_object" target="helpWin">more...</a>
</p>
</div>
</td>
</tr>

<tr class="odd paramSet">
<th>
Alignment&nbsp;View
</th>
<td>
<div class="fi">
<select name="ALIGNMENT_VIEW" defval="Pairwise" class="reset">
<option value="Pairwise" selected="selected">Pairwise</option>
<option value="PairwiseWithIdentities">Pairwise with dots for identities</option>
<option value="QueryAnchored">Query-anchored with dots for identities</option>
<option value="QueryAnchoredNoIdentities">Query-anchored with letters for identities</option>
<option value="FlatQueryAnchored">Flat query-anchored with dots for identities</option>
<option value="FlatQueryAnchoredNoIdentities">Flat query-anchored with letters for identities</option>
</select>
<a class="helplink hiding" title="help" id="alnViewHelp" href="#"><span>[?]</span></a>
<p toggle="alnViewHelp" class="helpbox hidden" id="hlp2">
Choose how to view alignments.
The default "pairwise" view shows how each subject sequence aligns
individually to the query sequence. The "query-anchored" view shows how
all subject sequences align to the query sequence. For each view type,
you can choose to show "identities" (matching residues) as letters or
dots.
<a href="http://www.ncbi.nlm.nih.gov/BLAST/blastcgihelp.shtml#alignment_view" target="helpWin">more...</a>
</p>
</div>
</td>
</tr>

<tr class="paramSet">
<th>Display</th>
<td class="cb">
<div class="fi">
<input name="SHOW_OVERVIEW" id="SHOW_OVERVIEW" class="cb reset" defval="checked" checked="checked" type="checkbox">
<label class="rb" for="SHOW_OVERVIEW">Graphical Overview</label>

<input name="SHOW_LINKOUT" id="SHOW_LINKOUT" class="cb reset" defval="checked" checked="checked" type="checkbox">
<label class="rb" for="SHOW_LINKOUT">Linkout</label>

<input name="GET_SEQUENCE" id="GET_SEQUENCE" class="cb reset" defval="checked" checked="checked" type="checkbox">
<label class="rb" for="GET_SEQUENCE">Sequence Retrieval</label>

<input name="NCBI_GI" id="NCBI_GI" class="cb reset" defval="unchecked" type="checkbox">
<label class="rb" for="NCBI_GI">NCBI-gi</label>

<input name="SHOW_CDS_FEATURE" id="SHOW_CDS_FEATURE" class="cb reset blastn" defval="unchecked" type="checkbox">
<label for="SHOW_CDS_FEATURE" class="blastn">CDS feature</label>
<a class="helplink hiding" title="help" id="displayHelp" href="#"><span>[?]</span></a>
<ul toggle="displayHelp" class="helpbox hidden" id="hlp3">
<li>Graphical Overview: Graphical Overview: Show graph of similar sequence regions aligned to  query.
<a href="http://www.ncbi.nlm.nih.gov/BLAST/blastcgihelp.shtml#show_overview" target="helpWin">more...</a>
</li>
<li>Database LinkOuts: Show links from matching sequences to entries in specialized NCBI databases.
<a href="http://www.ncbi.nlm.nih.gov/BLAST/blastcgihelp.shtml#show_linkout" target="helpWin">more...</a>
</li>
<li>Sequence Retrieval: Show buttons to download matching sequences.
<a href="http://www.ncbi.nlm.nih.gov/BLAST/blastcgihelp.shtml#get_sequence" target="helpWin">more...</a>
</li>
<li>NCBI-gi: Show NCBI gi identifiers.
<a href="http://www.ncbi.nlm.nih.gov/BLAST/blastcgihelp.shtml#ncbi_gi" target="helpWin">more...</a>
</li>
<li>CDS feature: Show annotated coding region and translation.
<a href="http://www.ncbi.nlm.nih.gov/BLAST/blastcgihelp.shtml#show_cds_feature" target="helpWin">more...</a>
</li>
</ul>
</div>
</td>
</tr>


<tr class="paramSet odd">
<th>

</th>
<td>
<div class="fi">
<label for="MASK_CHAR">Masking Character: </label>
<select name="MASK_CHAR" id="MASK_CHAR" class="reset" defval="2">
<option value="0">X for protein, n for nucleotide</option>
<option value="2" selected="selected">Lower Case</option>
</select>
<label for="MASK_COLOR">Masking Color:</label>
<select name="MASK_COLOR" id="MASK_COLOR" class="reset" defval="1">
<option value="0">Black
</option>

<option value="1" selected="selected">Grey
</option>

<option value="2">Red
</option>

</select>
<a class="helplink hiding" title="help" id="maskingHelp" href="#"><span>[?]</span></a>
<ul toggle="maskingHelp" class="helpbox hidden" id="hlp4">
<li>Masking Character: Display masked (filtered) sequence regions as
lower-case or as specific letters (N for nucleotide, P for protein).
</li>
<li>Masking Color: Display masked sequence regions in the given color.</li>
</ul>
</div>
</td>
</tr>


<tr class="paramSet">
<th>
Limit results
</th>
<td>
<div class="fi">
<label for="DESCRIPTIONS">Descriptions:</label>
<select name="DESCRIPTIONS" id="DESCRIPTIONS" class="reset" defval="100">
<option value="0">0</option>
<option value="10">10</option>
<option value="50">50</option>
<option value="100" selected="selected">100</option>






</select>

<label for="NUM_OVERVIEW">Graphical overview:</label>
<select name="NUM_OVERVIEW" id="NUM_OVERVIEW" class="reset" defval="100">
<option value="0">0</option>
<option value="10">10</option>
<option value="50">50</option>
<option value="100" selected="selected">100</option>



</select>

<label for="ALIGNMENTS">Alignments:</label>
<select name="ALIGNMENTS" id="ALIGNMENTS" class="reset" defval="100">
<option value="0">0</option>
<option value="10">10</option>
<option value="50">50</option>
<option value="100" selected="selected">100</option>






</select>
<a class="helplink hiding" title="help" id="numHelp" href="#"><span>[?]</span></a>
<ul toggle="numHelp" class="helpbox hidden" id="hlp5">
<li>Descriptions: Show short descriptions for up to the given number of  sequences.</li> 
<li>Alignments:  Show alignments for up to the given number of sequences, in order of statistical significance.</li>
</ul>
</div>
</td>

</tr><tr class="paramSet odd">
<th>

</th>
<td>
<div class="fi">
<label for="FORMAT_ORGANISM">Organism</label>
<span class="instr">Type common name, binomial, taxid, or group name. Only 20 top taxa will be shown.</span>
<input autocomplete="off" name="FORMAT_ORGANISM" size="55" id="qorganism" sgdict="taxids_sg" value="" sgstyle="SG_green" class="reset orgHint">
<a class="helplink hiding" title="help" id="organismHelp" href="#"><span>[?]</span></a>
<p toggle="organismHelp" class="helpbox hidden" id="hlp6">
Show only sequences from the given organism.
</p>
</div>
</td>
</tr>

<tr class="paramSet">
<th>
</th>
<td>
<div class="fi">
<label for="FORMAT_EQ_TEXT">Entrez query:</label>
<input name="FORMAT_EQ_TEXT" id="FORMAT_EQ_TEXT" size="60" value="" class="reset" type="text">
<a class="helplink hiding" title="help" id="entrezHelp" href="#"><span>[?]</span></a>
<p toggle="entrezHelp" class="helpbox hidden" id="hlp7">
Show only those sequences that match the given Entrez query.
<a href="http://www.ncbi.nlm.nih.gov/BLAST/blastcgihelp.shtml#limit_result" target="helpWin">more...</a>
</p>
</div>
</td>
</tr>



<tr class="paramSet odd">
<th>

</th>
<td>
<div class="fi">
<label for="EXPECT_LOW">Expect Min:</label> <input name="EXPECT_LOW" id="EXPECT_LOW" size="10" value="" class="reset" type="text">
<label for="EXPECT_HIGH">Expect Max:</label> <input name="EXPECT_HIGH" id="EXPECT_HIGH" size="10" value="" class="reset" type="text">
<a class="helplink hiding" title="help" id="expectHelp" href="#"><span>[?]</span></a>
<p toggle="expectHelp" class="helpbox hidden" id="hlp8">
Show only sequences with expect values in the given range.
<a href="http://www.ncbi.nlm.nih.gov/BLAST/blastcgihelp.shtml#expect_range" target="helpWin">more...</a>
</p>
</div>
</td>
</tr>
<tr class="psiBlast paramSet">
<th>
Format for
</th>
<td>
<div class="fi">
<input name="RUN_PSIBLAST_FORM" id="RUN_PSIBLAST" class="cb psiBlast" type="checkbox">
<label class="rb psiBlast" for="RUN_PSIBLAST">PSI-BLAST</label>
<label for="I_THRESH">with inclusion threshold:</label>
<input name="I_THRESH" id="I_THRESH" size="10" value="" defval="0.005" type="text">
<a class="helplink hiding" title="help" id="psiHelp" href="#"><span>[?]</span></a>
<ul toggle="psiHelp" class="helpbox hidden" id="hlp9">
<li>Format for PSI-BLAST: The Position-Specific Iterated BLAST
(PSI-BLAST) program performs iterative searches with a protein query,
in which sequences found in one round of search are used to build a
custom score model for the next round.</li>
<a href="http://www.ncbi.nlm.nih.gov/BLAST/blastcgihelp.shtml#psiblast" target="helpWin">more...</a>
<li>Inclusion Threshold: This sets the statistical significance threshold for including a sequence in the model used 
by PSI-BLAST to create the PSSM on the next iteration.</li> 
</ul>
</div>
</td>
</tr>
</tbody></table>


<input name="ViewReport" value="ViewReport" type="hidden">
<input name="REFORMAT_REPORT" value="shown" type="hidden">
<input name="RID" value="T25E2RUR11N" type="hidden">
<input name="CDD_RID" value="" type="hidden">
<input name="CDD_SEARCH_STATE" value="" type="hidden">

<input name="STEP_NUMBER" value="" id="stepNumber" type="hidden">
<input name="CMD" value="Get" type="hidden">
<input name="FORMAT_EQ_OP" value="AND" type="hidden">
<input name="RESULTS_PAGE_TARGET" id="resPageTarget" value="" type="hidden">
<!-- <input name="QUERY_INFO" type="hidden" value="1smd AMY1_HUMAN (488 letters)" /> -->	
<!-- <input name="ENTREZ_QUERY" type="hidden" value="" /> -->	
<input name="QUERY_INDEX" value="0" type="hidden">
<!-- Those params are set in the template (blastn.dat, blastp.dat etc. -->
<input name="BLAST_PROGRAMS" value="blastp" type="hidden">
<input name="PAGE" value="Proteins" type="hidden">
<input name="PROGRAM" value="blastp" type="hidden">
<input name="MEGABLAST" value="" type="hidden">
<input name="RUN_PSIBLAST" value="" type="hidden">
<input name="BLAST_SPEC" id="blastSpec" value="blast2seq" type="hidden">


<input name="QUERY" value="" type="hidden">
<input name="JOB_TITLE" value="1smd AMY1_HUMAN" type="hidden">
<input name="QUERY_TO" value="" type="hidden">
<input name="QUERY_FROM" value="" type="hidden">
<input name="EQ_TEXT" value="" type="hidden">
<input name="ORGN" value="" type="hidden">
<input name="EQ_MENU" value="" type="hidden">
<input name="PHI_PATTERN" value="" type="hidden">
<input name="EXPECT" value="10" type="hidden">									
<input name="DATABASE" value="nr" type="hidden">
<input name="GENETIC_CODE" value="" type="hidden">
<input name="WORD_SIZE" value="3" type="hidden">
<input name="MATCH_SCORES" value="" type="hidden">			
<input name="MATRIX_NAME" value="BLOSUM62" type="hidden">				
<input name="GAPCOSTS" value="11 1" type="hidden">
<input name="MAX_NUM_SEQ" id="maxNumSeq" value="100" type="hidden">					
<input name="COMPOSITION_BASED_STATISTICS" value="2" type="hidden">			
<input name="NEWWIN" value="" type="hidden">
<input name="SHORT_QUERY_ADJUST" value="on" type="hidden">
<input name="FILTER" value="" type="hidden">
<input name="REPEATS" value="repeat_9606" type="hidden">
<input name="ID_FOR_PSSM" value="" type="hidden">
<!-- PSSM -->
<input name="LCASE_MASK" value="" type="hidden">
<input name="TEMPLATE_TYPE" value="" type="hidden">
<input name="TEMPLATE_LENGTH" value="" type="hidden">
<input name="I_THRESH" value="" type="hidden">

<input name="ADJUSTED_FOR_SHORT_QUERY" value="" type="hidden">
<input name="MIXED_QUERIES" value="" type="hidden">

</form>
                <form name="FormatDownload" id="FormatDownload" class="resbox hidden" toggle="showDownload">
<table class="options dwnl">
<tbody><tr><th class="frh" colspan="6">Download </th></tr>
<tr>
<td id="tdal">
<div class="dnHeader">Alignment</div>
<div class="dnLinks">
<a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?RESULTS_FILE=on&amp;RID=T25E2RUR11N&amp;FORMAT_TYPE=Text&amp;FORMAT_OBJECT=Alignment&amp;CMD=Get">Text</a>
<a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?RESULTS_FILE=on&amp;RID=T25E2RUR11N&amp;FORMAT_TYPE=XML&amp;FORMAT_OBJECT=Alignment&amp;CMD=Get">XML</a>
<a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?RESULTS_FILE=on&amp;RID=T25E2RUR11N&amp;FORMAT_TYPE=ASN.1&amp;FORMAT_OBJECT=Alignment&amp;CMD=Get">ASN.1</a>
<a id="hitText" href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?RESULTS_FILE=on&amp;RID=T25E2RUR11N&amp;FORMAT_TYPE=Text&amp;FORMAT_OBJECT=Alignment&amp;ALIGNMENT_VIEW=Tabular&amp;CMD=Get">Hit Table(text)</a>
<a id="hitCvs" href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?RESULTS_FILE=on&amp;RID=T25E2RUR11N&amp;FORMAT_TYPE=CSV&amp;FORMAT_OBJECT=Alignment&amp;ALIGNMENT_VIEW=Tabular&amp;CMD=Get">Hit Table(csv)</a>
</div>
</td>
<td class="dnl">
<div class="dnHeader">Search Strategies</div>
<div class="dnLinks">
<a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=GetSaved&amp;RECENT_RESULTS=on&amp;RID=T25E2RUR11N&amp;DOWNLOAD_OPTIONS=true">ASN.1</a>
</div>
</td>
<td class="dnl">
<div class="dnHeader">Bioseq</div>
<div class="dnLinks">
<a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?RESULTS_FILE=on&amp;RID=T25E2RUR11N&amp;FORMAT_TYPE=ASN.1&amp;FORMAT_OBJECT=Bioseq&amp;CMD=Get">ASN.1</a>
</div>
</td>
<td class="dnl hidden" id="dnPSSM">
<div class="dnHeader">PSSM</div>
<div class="dnLinks">
<a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?RESULTS_FILE=on&amp;RID=T25E2RUR11N&amp;FORMAT_TYPE=Text&amp;FORMAT_OBJECT=PSSM&amp;CMD=Get">Text</a>
</div>
</td>
<td class="dnl hidden" id="dnPSSMPar">
<div class="dnHeader">PssmWithParameters</div>
<div class="dnLinks">
<a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?RESULTS_FILE=on&amp;RID=T25E2RUR11N&amp;FORMAT_TYPE=ASN.1&amp;FORMAT_OBJECT=PSSM_Scoremat&amp;CMD=Get">ASN.1</a>
</div>
</td>
<td>
<a class="helplink hiding" title="help" id="dnHelp" href="#"><span>[?]</span></a>
</td>
</tr>
</tbody></table>
<p toggle="dnHelp" class="helpbox hidden" id="hlp10">
These options control formatting of alignments in results pages. The
default is HTML, but other formats (including plain text) are available.
PSSM and PssmWithParameters are representations of Position Specific Scoring Matrices and are only available for PSI-BLAST. 
The Advanced view option allows the database descriptions to be sorted by various indices in a table.
<a href="http://www.ncbi.nlm.nih.gov/BLAST/blastcgihelp.shtml#format_object" target="helpWin">more...</a>
</p>
</form>
                <form action="Blast.cgi" enctype="application/x-www-form-urlencoded" method="post" name="overview0">
                <input name="ALIGNMENTS" value="100" type="hidden"><input name="ALIGNMENT_VIEW" value="Pairwise" type="hidden"><input name="BL2SEQ" value="on" type="hidden"><input name="BLAST_INIT" value="blast2seq" type="hidden"><input name="BLAST_PROGRAMS" value="blastp" type="hidden"><input name="BLAST_SPEC" value="blast2seq" type="hidden"><input name="CDD_SEARCH" value="on" type="hidden"><input name="CLIENT" value="web" type="hidden"><input name="COMPOSITION_BASED_STATISTICS" value="2" type="hidden"><input name="DATABASE" value="nr" type="hidden"><input name="DESCRIPTIONS" value="100" type="hidden"><input name="EQ_OP" value="AND" type="hidden"><input name="EXPECT" value="10" type="hidden"><input name="FORMAT_OBJECT" value="Alignment" type="hidden"><input name="FORMAT_TYPE" value="HTML" type="hidden"><input name="FULL_DBNAME" value="n/a" type="hidden"><input name="GAPCOSTS" value="11 1" type="hidden"><input name="GET_SEQUENCE" value="on" type="hidden"><input name="JOB_TITLE" value="1smd AMY1_HUMAN" type="hidden"><input name="LAYOUT" value="OneWindow" type="hidden"><input name="MASK_CHAR" value="2" type="hidden"><input name="MASK_COLOR" value="1" type="hidden"><input name="MATRIX_NAME" value="BLOSUM62" type="hidden"><input name="MAX_NUM_SEQ" value="100" type="hidden"><input name="NUM_DIFFS" value="0" type="hidden"><input name="NUM_OPTS_DIFFS" value="0" type="hidden"><input name="NUM_OVERVIEW" value="100" type="hidden"><input name="OLD_BLAST" value="false" type="hidden"><input name="PAGE" value="Proteins" type="hidden"><input name="PAGE_TYPE" value="BlastSearch" type="hidden"><input name="PROGRAM" value="blastp" type="hidden"><input name="QUERY_INFO" value=" 1smd AMY1_HUMAN (488 letters)" type="hidden"><input name="QUERY_LENGTH" value="488" type="hidden"><input name="REPEATS" value="repeat_9606" type="hidden"><input name="RID" value="T25E2RUR11N" type="hidden"><input name="RTOE" value="1222" type="hidden"><input name="SEARCH_DB_STATUS" value="43" type="hidden"><input name="SELECTED_PROG_TYPE" value="blastp" type="hidden"><input name="SERVICE" value="plain" type="hidden"><input name="SHORT_QUERY_ADJUST" value="on" type="hidden"><input name="SHOW_LINKOUT" value="on" type="hidden"><input name="SHOW_OVERVIEW" value="on" type="hidden"><input name="USER_DEFAULT_MATRIX" value="3" type="hidden"><input name="USER_DEFAULT_PROG_TYPE" value="blastp" type="hidden"><input name="USER_TYPE" value="2" type="hidden"><input name="WORD_SIZE" value="3" type="hidden"><input name="_PGR" value="0" type="hidden"><input name="db" value="protein" type="hidden"><input name="stype" value="protein" type="hidden">
                <input name="CMD" value="Get" type="hidden">
                <p id="qb"><!--
QBlastInfoBegin
	Status=READY
QBlastInfoEnd
--></p>
<div id="page">

<div id="summary" class="">
<div id="ht">Blast 2 sequences</div>
<h1>1smd AMY1_HUMAN <span id="hdTitle" class="shownInl"></span></h1>
<div>
<div id="multipleSeqs" class="hidden">
<label for="queryList">Results for:</label>
<select id="queryList" name="queryList">
<option value="0" selected="selected" class="">lcl|36839 1smd AMY1_HUMAN(488aa)</option>
</select>
<a class="helplink hiding" title="help" id="queryHelp" href="#">What's this?</a>
</div><!--/#multipleSeqs"-->

<p id="queryInfo" toggle="queryHelp" class="helpbox hidden">
Your BLAST job specified more than one input sequence.
This box lets you choose which input sequence to show BLAST results for.
</p>
</div>
<div id="summarylists">
<div id="querysummary">
<dl>
<dt>Query ID</dt>
<dd>lcl|36839</dd>
<dt>Description</dt>
<dd>1smd AMY1_HUMAN</dd>
<dt>Molecule type</dt>
<dd>amino acid</dd>
<dt>Query Length</dt>
<dd>488</dd>
</dl>
</div><!--/#querysummary-->

<div id="dbsummary">
<dl>
<dt>Subject ID</dt>
<dd>36841</dd>
<dt>Description</dt>
<dd>1bf2 ISOA_PSEAY <a href="#" title="Multiple subjects" class="resArrowLinkB hiding hidden" id="showDetails">See details</a></dd>
<dt>Molecule type</dt>
<dd>amino acid</dd>
<dt>Subject Length</dt>
<dd>572</dd> <!-- -- Database or subject info ---->
<dt>Program</dt>
<dd>BLASTP 2.2.19+ <a href="#" title="How to cite BLAST in publications" class="resArrowLinkB hiding hidden shown" id="showRefs">Citation</a></dd>
</dl>
</div><!--/#dbsummary-->
</div><!--/#summarylists -->

<div id="dbDetails" toggle="showDetails" class="helpbox hidden">
<table class="dbInfo">
<tbody><tr><th>Subject ID</th><th>Description</th><th>Length</th></tr>

</tbody></table> <!-- multiple databases or subjects table -->
</div>
<div id="refInfo" toggle="showRefs" class="helpbox hidden"> 
<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Retrieve&amp;list_uids=9254694&amp;dopt=Citation">Reference </a>
<div>Stephen F. Altschul, Thomas L. Madden, Alejandro A. Schäffer,
Jinghui Zhang, Zheng Zhang, Webb Miller, and David J. Lipman (1997),
"Gapped BLAST and PSI-BLAST: a new generation of protein database
search programs", Nucleic Acids Res. 25:3389-3402.</div><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Retrieve&amp;list_uids=16218944&amp;dopt=Citation">Reference  - compositional score matrix adjustment</a>
<div>Stephen F. Altschul, John C. Wootton, E. Michael Gertz, Richa
Agarwala, Aleksandr Morgulis, Alejandro A. Schäffer, and Yi-Kuo Yu
(2005) "Protein database searches using compositionally adjusted
substitution matrices", FEBS J. 272:5101-5109.</div>       
 </div>  <!-- refInfo -->
<!--<ul class="msg"><li class=""><p class=""></p></li></ul>-->

<div id="seealso">
<label>Other reports:</label>  
 <a href="#" title="Show statistics" class="resArrowLinkB hiding" id="showStat">Search Summary</a>
 <a id="tax" class="hidden shown" href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Get&amp;RID=T25E2RUR11N&amp;FORMAT_OBJECT=TaxBlast&amp;NCBI_GI=off&amp;DESCRIPTIONS=100&amp;ALIGNMENTS=&amp;FORMAT_BLOCK_ON_RESPAGE=Top&amp;MASK_COLOR=1&amp;MASK_CHAR=2" target="Taxonomy BLAST Results for T25E2RUR11N" title="See results arranged by taxonomy">[Taxonomy reports]</a>
    
   
  
 <!-- -->
</div> <!-- seealso -->
   
<div id="resStat" toggle="showStat" class="helpbox hidden">    
 <table id="params" class="dbInfo"> 
 <tbody><tr><th colspan="2">Search Parameters</th></tr>
 <tr><td>Program</td><td>blastp</td></tr><tr><td>Word size</td><td>3</td></tr><tr><td>Expect value</td><td>10</td></tr><tr><td>Hitlist size</td><td>100</td></tr><tr><td>Gapcosts</td><td>11,1</td></tr><tr><td>Matrix</td><td>BLOSUM62</td></tr><tr><td>Threshold</td><td>11</td></tr><tr><td>Composition-based stats</td><td>2</td></tr><tr><td>Filter string </td><td>F</td></tr><tr><td>Genetic Code</td><td>1</td></tr><tr><td>Window Size</td><td>40</td></tr> 
 </tbody></table> 
 <table id="Kappa" class="dbInfo show"> 
 <tbody><tr><th colspan="3">Karlin-Altschul statistics</th></tr> 
 <tr><td>Params</td><td>Ungapped</td><td>Gapped</td></tr>
 <tr><td>Lambda</td><td>0.319994</td><td>0.267</td></tr>
 <tr><td>K</td><td>0.138691</td><td>0.041</td></tr>
 <tr><td>H</td><td>0.458753</td><td>0.14</td></tr> 
 </tbody></table> 
  
<table class="dbInfo">
 <tbody><tr><th colspan="2">Results Statistics</th></tr>  
 <tr><td>Effective search space</td><td>243261</td></tr>
</tbody></table>
 
 <!-- Results statistics -->  
 </div>  <!-- resStat -->
</div><!--/#summary-->
</div><!--/#page-->
<!--<h2 class="" >Graphic Summary</h2>-->
<div id="viewRes">

<div id="grView" class="hidden shown">
<a href="#" class="arrowlink shown showing resArr" id="showGraphic">Graphic Summary</a>
<div id="graphicInfo" toggle="showGraphic" class="resBox shown" name="setToggle">


<div id="graphic" class="hidden shown">
<h3>Distribution of 7 Blast Hits on the Query Sequence
<a class="helplink hiding" title="help" id="grHelp" href="#"><span>[?]</span></a></h3>
<p id="ovrInfo" toggle="grHelp" class="helpbox hidden">
An overview of the database sequences aligned to the query sequence is
shown. The score of each alignment is indicated by one of five
different colors, which divides the range of scores into five groups.
Multiple alignments on the same database sequence are connected by a
striped line. <span class="hl">Mousing over</span> a hit sequence causes the definition and score to be shown in the window at the top, 
<span class="hl">clicking</span> on a hit sequence takes the user to the associated alignments.
New:
This graphic is an overview of database sequences aligned to the query sequence.
Alignments are color-coded by score, within one of five score ranges.  
Multiple alignments on the same database sequence are connected by a dashed line.
<span class="hl">Mousing</span> over an alignment shows the alignment definition and score in the box at the top. 
<span class="hl">Clicking</span> an alignment displays the alignment detail.
</p>
<center><input name="defline" size="85" value="Mouse over to see the defline, click to show alignments" type="text"><table bordercolordark="#0000FF" bordercolorlight="#0000FF" border="1" cellpadding="10" cellspacing="0"><tbody><tr><td align="left" valign="CENTER"><table border="0" cellpadding="0" cellspacing="1"><tbody><tr><td align="left" valign="CENTER"><img src="1smd_vs_1bf2_blast2sequences_files/white.gif" height="4" width="50"></td>
<td align="left" valign="CENTER"><img src="1smd_vs_1bf2_blast2sequences_files/score.gif" height="40" width="500"></td>
</tr></tbody></table>
<table border="0" cellpadding="0" cellspacing="1"><tbody><tr><td align="left" valign="CENTER"><img src="1smd_vs_1bf2_blast2sequences_files/query_no_scale.gif" height="10" width="550"></td></tr></tbody></table>
<table border="0" cellpadding="0" cellspacing="0"><tbody><tr><td align="left" valign="CENTER"><img src="1smd_vs_1bf2_blast2sequences_files/white.gif" height="4" width="51"></td>
<td align="left" valign="CENTER"><img src="1smd_vs_1bf2_blast2sequences_files/scale.gif" height="10" width="2"></td>
<td align="left" valign="CENTER"><img src="1smd_vs_1bf2_blast2sequences_files/white.gif" height="4" width="90"></td>
<td align="left" valign="CENTER"><img src="1smd_vs_1bf2_blast2sequences_files/scale.gif" height="10" width="2"></td>
<td align="left" valign="CENTER"><img src="1smd_vs_1bf2_blast2sequences_files/white.gif" height="4" width="90"></td>
<td align="left" valign="CENTER"><img src="1smd_vs_1bf2_blast2sequences_files/scale.gif" height="10" width="2"></td>
<td align="left" valign="CENTER"><img src="1smd_vs_1bf2_blast2sequences_files/white.gif" height="4" width="90"></td>
<td align="left" valign="CENTER"><img src="1smd_vs_1bf2_blast2sequences_files/scale.gif" height="10" width="2"></td>
<td align="left" valign="CENTER"><img src="1smd_vs_1bf2_blast2sequences_files/white.gif" height="4" width="90"></td>
<td align="left" valign="CENTER"><img src="1smd_vs_1bf2_blast2sequences_files/scale.gif" height="10" width="2"></td>
<td align="left" valign="CENTER"><img src="1smd_vs_1bf2_blast2sequences_files/white.gif" height="4" width="90"></td>
<td align="left" valign="CENTER"><img src="1smd_vs_1bf2_blast2sequences_files/scale.gif" height="10" width="2"></td>
</tr></tbody></table>
<table border="0" cellpadding="0" cellspacing="0"><tbody><tr><td align="left" valign="CENTER"><img src="1smd_vs_1bf2_blast2sequences_files/white.gif" height="4" width="50"></td>
<td align="left" valign="CENTER"><img src="1smd_vs_1bf2_blast2sequences_files/0.gif" height="13" width="10"></td>
<td align="left" valign="CENTER"><img src="1smd_vs_1bf2_blast2sequences_files/white.gif" height="4" width="72"></td>
<td align="left" valign="CENTER"><img src="1smd_vs_1bf2_blast2sequences_files/9.gif" height="13" width="10"></td>
<td align="left" valign="CENTER"><img src="1smd_vs_1bf2_blast2sequences_files/0.gif" height="13" width="10"></td>
<td align="left" valign="CENTER"><img src="1smd_vs_1bf2_blast2sequences_files/white.gif" height="4" width="72"></td>
<td align="left" valign="CENTER"><img src="1smd_vs_1bf2_blast2sequences_files/1.gif" height="13" width="10"></td>
<td align="left" valign="CENTER"><img src="1smd_vs_1bf2_blast2sequences_files/8.gif" height="13" width="10"></td>
<td align="left" valign="CENTER"><img src="1smd_vs_1bf2_blast2sequences_files/0.gif" height="13" width="10"></td>
<td align="left" valign="CENTER"><img src="1smd_vs_1bf2_blast2sequences_files/white.gif" height="4" width="62"></td>
<td align="left" valign="CENTER"><img src="1smd_vs_1bf2_blast2sequences_files/2.gif" height="13" width="10"></td>
<td align="left" valign="CENTER"><img src="1smd_vs_1bf2_blast2sequences_files/7.gif" height="13" width="10"></td>
<td align="left" valign="CENTER"><img src="1smd_vs_1bf2_blast2sequences_files/0.gif" height="13" width="10"></td>
<td align="left" valign="CENTER"><img src="1smd_vs_1bf2_blast2sequences_files/white.gif" height="4" width="62"></td>
<td align="left" valign="CENTER"><img src="1smd_vs_1bf2_blast2sequences_files/3.gif" height="13" width="10"></td>
<td align="left" valign="CENTER"><img src="1smd_vs_1bf2_blast2sequences_files/6.gif" height="13" width="10"></td>
<td align="left" valign="CENTER"><img src="1smd_vs_1bf2_blast2sequences_files/0.gif" height="13" width="10"></td>
<td align="left" valign="CENTER"><img src="1smd_vs_1bf2_blast2sequences_files/white.gif" height="4" width="62"></td>
<td align="left" valign="CENTER"><img src="1smd_vs_1bf2_blast2sequences_files/4.gif" height="13" width="10"></td>
<td align="left" valign="CENTER"><img src="1smd_vs_1bf2_blast2sequences_files/5.gif" height="13" width="10"></td>
<td align="left" valign="CENTER"><img src="1smd_vs_1bf2_blast2sequences_files/0.gif" height="13" width="10"></td>
</tr></tbody></table>
<br><table border="0" cellpadding="0" cellspacing="0"><tbody><tr><td align="left" valign="CENTER"><img src="1smd_vs_1bf2_blast2sequences_files/white.gif" height="4" width="50"></td>
<td align="left" valign="CENTER"><img src="1smd_vs_1bf2_blast2sequences_files/white.gif" height="4" width="30"></td>
<td align="left" valign="CENTER"><a href="#36841"><img onmouseout="document.overview0.defline.value='Mouse-over to show defline and scores, click to show alignments'" onmouseover="document.overview0.defline.value='36841 1bf2 ISOA_PSEAY S=16 E=3.8'" src="1smd_vs_1bf2_blast2sequences_files/black.gif" border="0" height="4" width="24"></a></td>
<td align="left" valign="CENTER"><img src="1smd_vs_1bf2_blast2sequences_files/grey.gif" height="1" width="13"></td>
<td align="left" valign="CENTER"><a href="#36841"><img onmouseout="document.overview0.defline.value='Mouse-over to show defline and scores, click to show alignments'" onmouseover="document.overview0.defline.value='36841 1bf2 ISOA_PSEAY S=31 E=7.2e-05'" src="1smd_vs_1bf2_blast2sequences_files/black.gif" border="0" height="4" width="30"></a></td>
<td align="left" valign="CENTER"><img src="1smd_vs_1bf2_blast2sequences_files/grey.gif" height="1" width="183"></td>
<td align="left" valign="CENTER"><a href="#36841"><img onmouseout="document.overview0.defline.value='Mouse-over to show defline and scores, click to show alignments'" onmouseover="document.overview0.defline.value='36841 1bf2 ISOA_PSEAY S=15 E=7.5'" src="1smd_vs_1bf2_blast2sequences_files/black.gif" border="0" height="4" width="29"></a></td>
<td align="left" valign="CENTER"><img src="1smd_vs_1bf2_blast2sequences_files/grey.gif" height="1" width="57"></td>
<td align="left" valign="CENTER"><a href="#36841"><img onmouseout="document.overview0.defline.value='Mouse-over to show defline and scores, click to show alignments'" onmouseover="document.overview0.defline.value='36841 1bf2 ISOA_PSEAY S=20 E=0.26'" src="1smd_vs_1bf2_blast2sequences_files/black.gif" border="0" height="4" width="51"></a></td>
<td align="left" valign="CENTER"><img src="1smd_vs_1bf2_blast2sequences_files/grey.gif" height="1" width="28"></td>
<td align="left" valign="CENTER"><a href="#36841"><img onmouseout="document.overview0.defline.value='Mouse-over to show defline and scores, click to show alignments'" onmouseover="document.overview0.defline.value='36841 1bf2 ISOA_PSEAY S=18 E=0.84'" src="1smd_vs_1bf2_blast2sequences_files/black.gif" border="0" height="4" width="23"></a></td>
<td align="left" valign="CENTER"><img src="1smd_vs_1bf2_blast2sequences_files/black.gif" height="6" width="1"></td>
<td align="left" valign="CENTER"><a href="#36841"><img onmouseout="document.overview0.defline.value='Mouse-over to show defline and scores, click to show alignments'" onmouseover="document.overview0.defline.value='36841 1bf2 ISOA_PSEAY S=14 E=9'" src="1smd_vs_1bf2_blast2sequences_files/black.gif" border="0" height="4" width="3"></a></td>
</tr></tbody></table>
<table border="0" cellpadding="0" cellspacing="0"><tbody><tr><td><img src="1smd_vs_1bf2_blast2sequences_files/white.gif" height="4" width="550"></td></tr></tbody></table>
</td></tr></tbody></table>
</center><hr>
</div><!--/#graphic-->
</div><!-- graphicInfo -->
</div>
<!--HITMATRIX_VIEW - only for bl2seq -->
<div id="hitMatrixView" class="">
<a href="#" class="arrowlink shown hiding resArr" id="showHitMatrix">Dot Matrix View</a>
<img id="htmb" class="shown" src="1smd_vs_1bf2_blast2sequences_files/hit_matrix.html">
<div id="hitMatrixInfo" toggle="showHitMatrix" class="resBox hidden" name="setToggle">



<h3>Plot of lcl|36839 vs 36841
<a class="helplink hiding" title="help" id="bl2grHelp" href="#"><span>[?]</span></a></h3>
<p id="bl2ovrInfo" toggle="bl2grHelp" class="helpbox hidden">
This dot matrix view shows regions of similarity based upon the BLAST
results. The query sequence is represented on the X-axis and the
numbers represent the bases/residues of the query. The subject is
represented on the Y-axis and again the numbers represent the
bases/residues of the subject. Alignments are shown in the plot as
lines. Plus strand and protein matches are slanted from the bottom left
to the upper right corner, minus strand matches are slanted from the
upper left to the lower right. The number of lines shown in the plot is
the same as the number of alignments found by BLAST.
</p>



<center id="bl2seImg" class="hidden">
<img id="hitmtImg" src="" imgsrc="http://blast.ncbi.nlm.nih.gov/hitmatrix/hit_matrix.cgi/hit_matrix.cgi?rid=T25E2RUR11N&amp;width=600&amp;height=300" border="0">
</center>
</div><!-- hitMatrixInfo -->
</div>
<!--<h2 class="">Descriptions</h2>-->
<div id="dscrView" class="hidden shown">
<a href="#" class="arrowlink shown showing resArr" id="showDescr">Descriptions</a>
<div id="descrInfo" toggle="showDescr" class="resBox shown" name="setToggle">
 <!-- legend to links to other resources -->



<div id="descriptions" class="blRes">
<pre>                                                                   Score     E
Sequences producing significant alignments:                       (Bits)  Value

lcl|36841  1bf2 ISOA_PSEAY                                         <a href="#36841">31.6</a>    7e-05

</pre>
</div><!--/#descriptions-->
</div><!-- descrInfo -->
</div>
<!-- this is the end tag for the <form in blastcgi templates -->

<!--<h2 class="">Alignments</h2>-->
<div id="alignView" class="hidden shown">
<a href="#" class="arrowlink shown showing resArr" id="showAlign">Alignments</a>
<span name="seqlinks" class="seqlinks shownInl">
<input name="selectAll" id="selectAllTop" seqsform="getSeqAlignment1" type="checkbox"> <label for="selectAllTop">Select All</label>
<a name="getSeqs" class="ln" href="#">Get selected sequences</a>
<a href="#" name="treeView" title="See a distance tree of these pairwise comparisons" class="lnt hidden shown">Distance tree of results</a>
</span>
<form class="getseq" method="post" seqsform="getSeqAlignment1" action="http://www.ncbi.nlm.nih.gov/sites/entrez" id="submitterTop">
<input name="db" value="Protein" type="hidden">
<input id="selSeqs" name="term" value="" type="hidden">
<input name="doptcmdl" value="docsum" type="hidden">
<input name="cmd" value="search" type="hidden">
</form>
<form class="getseq" method="post" seqsform="getSeqAlignment1" action="?request=page&amp;rid=T25E2RUR11N&amp;queryID=&amp;distmode=on&amp;entrezLim=&amp;ex=&amp;exl=&amp;exh=&amp;ns=" name="treesubmitterTop" id="treesubmitterTop" target="trv">
<input id="seqSet" name="sequenceSet" value="" type="hidden">
<input id="scrWidth" name="screenWidth" value="" type="hidden">
</form>
<div id="alignInfo" toggle="showAlign" class="resBox shown" name="setToggle">
<div id="alignments" class="blRes">
<pre><script src="1smd_vs_1bf2_blast2sequences_files/blastResult.js"></script>&gt;lcl|36841<a name="36841"></a> 1bf2 ISOA_PSEAY
Length=572

 Score = 31.6 bits (70),  Expect = 7e-05, Method: Compositional matrix adjust.
 Identities = 12/30 (40%), Positives = 18/30 (60%), Gaps = 0/30 (0%)

Query  66   GNEDEFRNMVTRCNNVGVRIYVDAVINHMC  95
            G   EF+ MV   +N G+++Y+D V NH  
Sbjct  92   GPTAEFQAMVQAFHNAGIKVYMDVVYNHTA  121


 Score = 20.0 bits (40),  Expect = 0.26, Method: Compositional matrix adjust.
 Identities = 15/51 (29%), Positives = 25/51 (49%), Gaps = 1/51 (1%)

Query  358  VTKEVTINPDTTCGNDWVCEHRWRQIRNMVNFRNVVDGQPFTNWYD-NGSN  407
            +T  VT + +   G+  + +   R   N +NF +V DG    + Y  NG+N
Sbjct  296  MTIYVTQDANDFSGSSNLFQSSGRSPWNSINFIDVHDGMTLKDVYSCNGAN  346


 Score = 18.1 bits (35),  Expect = 0.84, Method: Compositional matrix adjust.
 Identities = 9/23 (39%), Positives = 14/23 (60%), Gaps = 0/23 (0%)

Query  435  GLPAGTYCDVISGDKINGNCTGI  457
            GL   TY ++ SG++   + TGI
Sbjct  141  GLDNATYYELTSGNQYFYDNTGI  163


 Score = 16.2 bits (30),  Expect = 3.8, Method: Compositional matrix adjust.
 Identities = 8/24 (33%), Positives = 11/24 (45%), Gaps = 0/24 (0%)

Query  30   GGVQVSPPNENVAIHNPFRPWWER  53
            GG      + NVAI+   R +  R
Sbjct  221  GGYNFDAADSNVAINRILREFTVR  244


 Score = 15.0 bits (27),  Expect = 6.7, Method: Compositional matrix adjust.
 Identities = 8/27 (29%), Positives = 11/27 (40%), Gaps = 3/27 (11%)

Query  378  HRWRQIRNMVNFRNVVDGQPFTNWYDN  404
            + WR + N   +      Q F   YDN
Sbjct  137  YSWRGLDNATYYELTSGNQYF---YDN  160


 Score = 15.0 bits (27),  Expect = 7.5, Method: Compositional matrix adjust.
 Identities = 8/29 (27%), Positives = 11/29 (37%), Gaps = 0/29 (0%)

Query  274  EGWGFMPSDRALVFVDNHDNQRGHGAGGA  302
            + W + PSD       + D     G G A
Sbjct  349  QAWPYGPSDGGTSTNYSWDQGMSAGTGAA  377


 Score = 14.6 bits (26),  Expect = 9.0, Method: Compositional matrix adjust.
 Identities = 4/6 (66%), Positives = 6/6 (100%), Gaps = 0/6 (0%)

Query  456  GIKIYV  461
            GIK+Y+
Sbjct  108  GIKVYM  113



</pre>
</div><!--/#alignments-->
</div><!-- alignInfo -->
<span name="seqlinks" class="seqlinks shownInl">
<input name="selectAll" id="selectAllBot" seqsform="getSeqAlignment1" type="checkbox"> <label for="selectAllBot">Select All</label>
<a name="getSeqs" class="ln" href="#">Get selected sequences</a>
<a href="#" name="treeView" title="See a distance tree of these pairwise comparisons" class="lnt hidden shown">Distance tree of results</a>
</span></div><!-- alignView -->
</div><!-- viewRes -->
<form><!-- this is the start tag for the </form in blastcgi templates - needs to be changed-->
<!-- not used - verify -->
<!-- not used - verify -->


                </form>
				</form></div><!-- /#content -->
				<form action="Blast.cgi" enctype="application/x-www-form-urlencoded" method="post" name="reformat" id="reformat">				
				   <input name="QUERY_INFO" value="1smd AMY1_HUMAN (488 letters)" type="hidden">    				
				   <input name="ENTREZ_QUERY" value="" type="hidden">
                   <input name="CDD_RID" value="" type="hidden">
                   <input name="CDD_SEARCH_STATE" value="" type="hidden">
                   <input name="RID" value="T25E2RUR11N" type="hidden">
				   <input name="STEP_NUMBER" value="" type="hidden">
				   <input name="CMD" value="Web" type="hidden">				
				   <input name="PAGE_TYPE" value="BlastFormatting" type="hidden">
				
				   <!-- TO DO: test all of those changes -->				   	
				   <!-- Psi blast params  PSI_BLAST_PARAMS - commented- using forms[0] from fromatter> -->
				   <!-- Current Formatting options FORMATTING_OPTIONS- commented- using forms[0] from fromatter> -->
				   <!-- Current Search options CURR_SAVED_OPTIONS - commented- using forms[0] from fromatter> -->
                 </form>

                 <form action="Blast.cgi" enctype="application/x-www-form-urlencoded" method="post" name="setSavedOptions" id="setSavedOptions">					
					<input name="PAGE_TYPE" value="BlastSearch" type="hidden">
					<input name="VIEW_SEARCH" value="true" type="hidden">
					<input name="CMD" value="Web" type="hidden">					
					<!-- Current Search options CURR_SAVED_OPTIONS - commented- using forms[0] from fromatter> -->
				</form>
				
				<form action="Blast.cgi" enctype="application/x-www-form-urlencoded" method="post" name="saveOptions" id="saveOptions">					
					<input name="SAVE" value="true" type="hidden">					
					<input name="CMD" value="GetSaved" type="hidden">
					<!-- Current Search options CURR_SAVED_OPTIONS - commented- using forms[0] from formatter> -->
				</form>
        </div><!-- /#content-wrap -->

		    	
<div id="footer">
   <p id="orgns">
      <a href="http://www.ncbi.nlm.nih.gov/" title="National Center for Biotechnology Information">NCBI</a> |
      <a href="http://www.nlm.nih.gov/" title="National Library of Medicine">NLM</a> |
      <a href="http://www.nih.gov/" title="National Institutes of Health">NIH</a> |
      <a href="http://www.hhs.gov/" title="US Department of Health and Human Services">DHHS</a>
   </p>
   
   <p>
      <a href="http://ncbi.nlm.nih.gov/About/disclaimer.html" title="NCBI intellectual property statement">Copyright</a> | 
      <a href="http://ncbi.nlm.nih.gov/About/disclaimer.html#disclaimer" title="About liability, endorsements, external links, pop-up advertisements">Disclaimer</a> | 
      <a href="http://www.nlm.nih.gov/privacy.html" title="NLM privacy policy">Privacy</a> | 
      <a href="http://ncbi.nlm.nih.gov/About/accessibility.html" title="About using NCBI resources with assistive technology">Accessibility</a> |
      <a href="http://www.ncbi.nlm.nih.gov/About/glance/contact_info.html" title="How to get help, submit data, or provide feedback">Contact</a> |
      <a href="mailto:blast-help@ncbi.nlm.nih.gov" title="How to get help, submit data, or provide feedback">Send feedback</a>      
   </p>
</div>	
   </div><!--/#wrap-->
<script type="text/javascript" src="1smd_vs_1bf2_blast2sequences_files/nsuggest_loader.js"></script>
<script type="text/javascript" src="1smd_vs_1bf2_blast2sequences_files/utils.js"></script>
<script type="text/javascript" src="1smd_vs_1bf2_blast2sequences_files/toggle.js"></script>
<script type="text/javascript" src="1smd_vs_1bf2_blast2sequences_files/results.js"></script>
<script type="text/javascript" src="1smd_vs_1bf2_blast2sequences_files/blast.js"></script>
<script type="text/javascript" src="1smd_vs_1bf2_blast2sequences_files/format.js"></script>
<script type="text/javascript" src="1smd_vs_1bf2_blast2sequences_files/nsuggest_ajax.js"></script>
</body></html>